modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -105,6 +105,16 @@ All page names are validated:
 * Page names cannot contain these characters: `@` `*` `+` `/` `_` `\`
 * Page names cannot start with `.`
 
+The special page *Home* has the URL `pages/home` and the special page *Terms and conditions* has the URL `pages/.terms`.
```

Comment:
Is there a reason that these pages are "special"?

Reply:
Good point! Added explanation.